parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -220,7 +223,11 @@ if [[ -n "${MASTER_NODE}" ]]; then
     fi
     time_metric "WriteKubeConfig" writeKubeConfig
     if [[ -z "${COSMOS_URI}" ]]; then
-        time_metric "EnsureEtcd" ensureEtcd
+        if [ "$FULL_INSTALL_REQUIRED" = "true" ]; then
```

Comment:
why is this change needed? It breaks the assumption that you can override the etcd version in the VHD by specifying a custom version.

Reply:
`ensureEtcd` depends upon eth0 having the correct IP address. In 18.04-LTS + non-VHD eth0 does not have the correct address.

Reply:
what about the VHD scenario? with this change we would be skipping ensureEtcd if using VHD

Reply:
Yeah that's a bug will fix

Reply:
@jackfrancis this wasn't fixed before the PR merged